src/ui/components/shared/Modals/PassportDismissModal.tsx

```diff
@@ -49,16 +48,10 @@ function PassportDismissModal({
         </DialogDescription>
         <DialogActions>
           <div className={styles.buttonContainer}>
-            <PrimaryButton color="blue" onClick={confirmDismiss}>
-              OK
-            </PrimaryButton>
-            <SecondaryButton
-              color="gray"
```

Comment:
This is another "gray" button. Gray buttons means disabled in our app. I've replaced this with the outline variant.